unitOneAssessmentPractice.js

```diff
@@ -1,138 +1,227 @@
 let assert = require('assert')
-
 // Question One:
-
 // Write a function called double that doubles a number
-
 // Uncomment out the next line to test your solution
-// runQ1Tests()
-
+const double = (num) => {
+  return num * 2
+}
+runQ1Tests()
 // Question Two:
-
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
-
 // Uncomment out the next line to test your solution
-// runQ2Tests()
-
+const containsSeven = (num) => {
+  let str = num + "";
+  return str.includes(7) ? true : false
```

Comment:
You don't need the ternary, simply adding str.includes('7') would be enough since the return value of includes is already a boolean